Firebase.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
```

Comment:
Curious if we know why the `Firebase.podspec` wasn't pushed to SpecsStaging? I just did a log-only run of firebase-releaser and it logs that `Firebase.podspec` is pod repo push'd.

Reply:
Good question and good catch - this is because when you run it it goes through one by one and will stop when one fails. One did fail and I was handling the rest manually. I knew it wasn't published yet but was trying to do multiple things at once (it was pushing while I opened the PR). Shouldn't happen when the tools are working as expected and all pods pass.